pubspec.lock

```diff
@@ -7,14 +7,14 @@ packages:
       name: _fe_analyzer_shared
```

Comment:
pubspec.lock shouldn't be committed (and should be in .gitignore)

Reply:
removed and added to .gitignore